main.go

```diff
@@ -27,20 +28,18 @@ func main() {
 
 	if len(os.Args) <= 1 {
 		plugin.FlagSet.Usage()
-		check.Exit(check.Unknown, "No arguments given")
+		check.ExitError(errors.New("No arguments given"))
 	}
 
 	err := config.Validate()
 	if err != nil {
 		check.ExitError(err)
 	}
 
-	plugin.SetupLogging()
-
 	err = config.Run()
 	if err != nil {
 		check.ExitError(err)
 	}
 
-	check.Exit(check.OK, "done")
+	check.Exitf(check.OK, "done")
```

Comment:
You can use `check.ExitRaw` it'll also take care of formatting the output.

Reply:
will do